### ISSUE TITLE
[Bug] Mod problem/button problem/version problem

### ISSUE THREAD

**Issue by danzheng666** [PojavLauncherTeam/PojavLauncher_iOS#582]
### Describe the bug

1. I try to open a higher version of forge, but when I enter the game, it shows "Exit with error code 1"
2. I tried to add some mods in versions 1.18.2, 1.16.5, and 1.12.2, but when I entered the game, I got stuck at "loading terrain"
3. I try to change to "Simplified Chinese" language in version 1.12.2, although it is displayed as selected, but I can't click "Completed"; change to the clicked language
4. I tried to change an option of "Menu Mapping" to "None" in the "Controller Configuration" interface, but it crashed
5. Can't call out the keyboard with the handle

### The log file and images/videos

[latestlog.old.txt](https://github.com/PojavLauncherTeam/PojavLauncher_iOS/files/11841168/latestlog.old.txt)
This text is about the high version forge
Other issues I don't know how happened and are not documents.

### Steps To Reproduce

```markdown
（1) 1. Open PojavLauncher
        2.Open “1.20.1-forge-47.0.18”
        3. After entering the game, the screen is black, then an error log appears, and then it shows "Exit with error code 1"
  (2) 1. enter the game
        2. create new world
        3. Stuck on "Loading terrain"
  (3) 1. enter the game
        2. language
        3. Switch to "Simplified Chinese" and get stuck, and enter the game again without changing
  (4) 1. Enter "handle configuration"
        2. Slide to "Menu Map"
        3. Modify a mapping to "None", from now on, as long as you slide to the "None" position, it will flash back
  (5) 1. enter the world
        2. Exit the chat bar
        3. Unable to call out the keyboard, can only manually click "Keyboard"
```

### Expected Behavior

I hope the author fixes these bugs and adds more button options (such as sliding)

### Platform

```markdown
- Device model: iPad Pro (11-inch) (2nd generation) (TrollStore) 
- iOS version: 15.2
- PojavLauncher version: 2.2
```

### Anything else?

Nothing else

**Comment by khanhduytran0** [comment]
1. https://github.com/PojavLauncherTeam/PojavLauncher_iOS/commit/78c378d fixed Forge 1.20.1
2. Need latestlog.txt
3. Are you using direct touch instead of virtual mouse?
4. Unknown. Send PojavLauncher analytics data.
5. I don’t understand what “call out the keyboard with the handle” means there. Did you mean hiding keyboard after exiting chat box?

For all those issue, first try updating to [latest build](https://github.com/PojavLauncherTeam/PojavLauncher_iOS/actions/runs/5355986510) and then try everything again.

**Comment by GaucovyAero** [comment]
stale, closing
